x-pack/plugins/apm/public/components/app/RumDashboard/RumHome.tsx

```diff
@@ -58,31 +59,38 @@ export function RumHome() {
         }
       : undefined;
 
+  const isLoading = status === 'loading';
+
   return (
-    <CsmSharedContextProvider>
-      <PageTemplateComponent
-        noDataConfig={noDataConfig}
-        pageHeader={
-          isXXL
-            ? {
-                pageTitle: i18n.translate('xpack.apm.ux.overview', {
-                  defaultMessage: 'Dashboard',
-                }),
-                rightSideItems: [
-                  <DatePicker />,
-                  <div style={envStyle}>
-                    <UxEnvironmentFilter />
-                  </div>,
-                  <UserPercentile />,
-                  <WebApplicationSelect />,
-                ],
-              }
-            : { children: <PageHeader /> }
-        }
-      >
-        <RumOverview />
-      </PageTemplateComponent>
-    </CsmSharedContextProvider>
+    <Fragment>
+      {isLoading && <EmptyStateLoading />}
```

Comment:
Wondering if we should only be showing this loading state for the UX content, instead of the entire page. Currently we are hiding everything, including the sidebar. I think showing the sidebar can be helpful for the user, especially if the loading is taking a long time and they want to navigate away from the screen easily.

Reply:
This loading is only valid for initial hasData call, rest of the loading calls for other content will remain in place.

Reply:
I understand, I just wonder if we should be hiding the entire page for this call, instead of just hiding the UX content, since the hasData call is specific to UX content.

Reply:
got it , i have imoroved it now.